fuse_constraints/include/fuse_constraints/absolute_orientation_3d_stamped_constraint.h

```diff
@@ -40,8 +40,8 @@
 #include <fuse_core/serialization.hpp>
 #include <fuse_core/uuid.hpp>
 #include <fuse_variables/orientation_3d_stamped.h>
-#include <geometry_msgs/PoseWithCovariance.h>
-#include <geometry_msgs/Quaternion.h>
+#include <geometry_msgs/msg/PoseWithCovariance.hpp>
```

Comment:
I think this is actually `geometry_msgs/msg/pose_with_covariance.hpp`. As far as I know all generated headers in ROS 2 are lowercase with underscores, so same comment throughout the PR

Reply:
It also looks like `PoseWithCovariance` is unused in this file, so this include could be removed.